### PR TITLE
Illia12.02

### PR DESCRIPTION
**Pull request by Ilya8490**

### CODE REVIEW

File: controllers/authController.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  res.status(status).json({
+    success: true,
+    status,
+    user,
+  });
+};
+
+//! Controllers
+export const signup = async (req, res, next) => {
```
Comment (selmaatasert) [inline controllers/authController.js:39]: You already create user(sign up) in userController. I think you can remove this method.

File: middlewares/validateSanitize.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { check, validationResult } from "express-validator";
+import createError from "http-errors";
+
+const validateSanitize = [
```
Comment (selmaatasert) [inline middlewares/validateSanitize.js:4]: Our signUp form We have dateOfBirth and telephone number. Can you add these validations too

File: routes/loginRouter.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import { signup, login, logout, protect } from '../controllers/authController.js';
```
Comment (selmaatasert) [inline routes/loginRouter.js:2]: we dont use signup, logout and protect in this file.

File: routes/usersRouter.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+import validateSanitize from "../middlewares/validateSanitize.js";
+import { login } from "../controllers/authController.js";
+
+const router = express.Router();
+
+router.route("/")
+  .get(usersController.getAllUsers) 
+  .post(validateSanitize, usersController.createUser); 
+
+router.route("/signup")
```
Comment (selmaatasert) [inline routes/usersRouter.js:12]: You already have signUp router. You can remove this .

File: routes/usersRouter.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  .post(validateSanitize, usersController.createUser); 
+
+router.route("/login")
```
Comment (selmaatasert) [inline routes/usersRouter.js:15]: you already have login router. You can remove this.

**Review by selmaatasert — COMMENTED** [review]
Look good, I have some small comments.

File: routes/loginRouter.js (hunk 3 above)
Comment (Ilya8490) [reply routes/loginRouter.js:2]: removed

File: controllers/authController.js (hunk 1 above)
Comment (Ilya8490) [reply controllers/authController.js:39]: done

**Pull request merged**